### PR TITLE
side bar design change + copy change for entire wizard

### PR DESCRIPTION
**Pull request by annielovesjs**
Home page:
- changed "Welcome to WTS " to "create your web  app in seconds"

frameworks & azure:
- changed details to learn more

pages:
- changed details to preview

Azure Services Page:
- changed styling of modal + made it more responsive 

Right side bar:
- changed styling
- added icons for hide, edit, and services 
- replace send feedback button with report an issue link

How to test:
- check the text
- it should look like this when you build it:
forms
rightsidebar

### CODE REVIEW

File: src/client/src/containers/About/index.tsx [hunk 1]
```diff
@@ -35,21 +39,28 @@ const About = ({ versions, intl }: Props) => {
...
           target="_blank"
           href={WEB_TEMPLATE_STUDIO_LINKS.REPO}
           onKeyUp={keyUpHandler}
         >
-          Web Template Studio
+          {formatMessage(messages.visitRepo)}
+        </a>
+        <a
+          className={styles.link}
+          href="https://github.com/Microsoft/WebTemplateStudio/issues"
```
Comment (streetcoder123) [inline src/client/src/containers/About/index.tsx]: Can we have `target="_blank"` here. It won't matter on the extension but it is more intuitive to test on browser.

Comment (annielovesjs) [reply src/client/src/containers/About/index.tsx]: Okay, I added it :)

File: src/client/src/containers/About/index.tsx [hunk 2]
```diff
@@ -35,21 +39,29 @@ const About = ({ versions, intl }: Props) => {
...
           target="_blank"
           href={WEB_TEMPLATE_STUDIO_LINKS.REPO}
           onKeyUp={keyUpHandler}
         >
-          Web Template Studio
+          {formatMessage(messages.visitRepo)}
+        </a>
+        <a
+          className={styles.link}
+          target="_blank"
```
Comment (streetcoder123) [inline src/client/src/containers/About/index.tsx:54]: Add `rel="noreferrer noopener"` for tslint whenever you use `target="_blank`. Thanks @Tanya0609 :)

**Review by streetcoder123 — DISMISSED** [review]

File: src/client/src/containers/About/index.tsx (hunk 2 above)
Comment (annielovesjs) [reply src/client/src/containers/About/index.tsx:54]: Just added :)

File: src/client/src/containers/About/index.tsx [hunk 3]
```diff
@@ -35,21 +39,31 @@ const About = ({ versions, intl }: Props) => {
...
+          rel="noreferrer noopener"
+          href="https://github.com/Microsoft/WebTemplateStudio/issues"
```
Comment (jsondoo) [inline src/client/src/containers/About/index.tsx]: Should we use a constant for this link? We can do it in a different PR if u want

**Review by jsondoo — DISMISSED** [review]

File: src/client/src/containers/About/index.tsx (hunk 3 above)
Comment (annielovesjs) [reply src/client/src/containers/About/index.tsx]: yeah I have to do something else with the sidebar so I will address that in that pr instead since I need this one to go in first

File: src/client/src/components/DraggableSidebarItem/index.tsx [hunk 4]
```diff
@@ -43,6 +49,8 @@ const DraggableSidebarItem = ({
 }: {
   page?: ISelected;
   text?: string;
+  azureFunctionsSvg?: boolean;
```
Comment (piotrmark) [inline src/client/src/components/DraggableSidebarItem/index.tsx]: NIT: since this is a boolean, can we name it isAzureFunc... or hasAzureFunc...
Same for cosmosDBSvg

File: src/client/src/components/DraggableSidebarItem/styles.module.css [hunk 5]
```diff
@@ -47,6 +47,10 @@
   margin-left: 10px;
 }
 
+.pagesTextContainerServices {
```
Comment (piotrmark) [inline src/client/src/components/DraggableSidebarItem/styles.module.css]: Is this class used somewhere?

File: src/client/src/components/DraggableSidebarItem/styles.module.css [hunk 6]
```diff
@@ -114,11 +118,15 @@
   outline: none;
   font-size: 1em;
   box-sizing: border-box;
-  background-color: var(--vscode-input-background);
+  background-color: var(--vscode-editor-background);
   color: var(--vscode-editor-foreground);
   border: none;
 }
 
+.servicesInput {
```
Comment (piotrmark) [inline src/client/src/components/DraggableSidebarItem/styles.module.css]: And this one?

File: src/client/src/components/SelectableCard/index.tsx [hunk 7]
```diff
@@ -121,8 +126,16 @@ const SelectableCard = ({
...
+              isPagesSelection
+                ? detailsConfig.preview.id
+                : detailsConfig.learnMore.id
+            }
+            defaultMessage={
+              isPagesSelection
+                ? detailsConfig.preview.default
+                : detailsConfig.learnMore.default
+            }
           />
```
Comment (piotrmark) [inline src/client/src/components/SelectableCard/index.tsx:139]: Maybe we can change it to check condition only once and get rid of detailsConfig?
```isPagesSelection ? <FormattedMessage id="selectableCard.preview" ...> : <FormattedMessage id="selectableCard.details" ...>```

File: src/client/src/containers/About/index.tsx (hunk 3 above)
Comment (piotrmark) [reply src/client/src/containers/About/index.tsx]: I think constant already exists, right? :)

File: src/client/src/components/DraggableSidebarItem/index.tsx (hunk 4 above)
Comment (annielovesjs) [reply src/client/src/components/DraggableSidebarItem/index.tsx]: yup! I just changed it

File: src/client/src/components/DraggableSidebarItem/styles.module.css (hunk 5 above)
Comment (annielovesjs) [reply src/client/src/components/DraggableSidebarItem/styles.module.css]: Oh yeah this was from an old class that I deleted but forgot to remove the styling for. Good catch!

Comment (annielovesjs) [reply src/client/src/components/DraggableSidebarItem/styles.module.css]: same with the one on the bottom - I will remove both

File: src/client/src/components/SelectableCard/index.tsx (hunk 7 above)
Comment (annielovesjs) [reply src/client/src/components/SelectableCard/index.tsx:139]: I was thinking of doing that initially but I thought that it would be more organized to store it config object especially in case we needed to add a third option later

Comment (piotrmark) [reply src/client/src/components/SelectableCard/index.tsx:139]: Yup, if we consider adding more options, it makes sense :)

**Review by piotrmark — DISMISSED** [review]

**Review by piotrmark — APPROVED** [review]

**Review by streetcoder123 — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]
looks good, i will update app service form after
